include/NovelRT/Experimental/Input/NovelKey.h

```diff
@@ -0,0 +1,34 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_EXPERIMENTAL_INPUT_NOVELKEY_H
+#define NOVELRT_EXPERIMENTAL_INPUT_NOVELKEY_H
+
+#ifndef NOVELRT_EXPERIMENTAL_INPUT_H
+#error NovelRT does not support including types explicitly by default. Please include Input.h instead for the Input namespace subset.
+#endif
+
+namespace NovelRT::Experimental::Input
+{
+    class NovelKey
+    {
+    private:
+        int32_t _pairedKey;
```

Comment:
Will this ever be negative?

Reply:
It will only be negative if the key is not paired as otherwise it's set to a specific key identifier from the impl. specific input device provider. (i.e. GLFW's key enum)